src/modules/site-v2/base/views/strains.py

```diff
@@ -303,7 +303,7 @@ def order_page_index():
   flash(Markup("<strong>Please note:</strong> although the site is currently able to accept orders, orders will <u>not ship</u> until Fall 2023."), category="warning")
 
   if not user and not cart_id:
-    cartItems = []
+    users_cart = []

```

Comment:
Hmm... I see what you're doing here, setting `users_cart` to an object with `len == 0` so it triggers the check below (line 316).

If possible, though, it's good practice to keep all uses of a variable the same type, for code clarity.  In this case, `users_cart` is assigned a `list` in the first branch of the `if` statement, and a `Cart` in the second branch, which makes it harder to reason about in later lines of code.

(In your specific case, you nullify the difference immediately, so this solution works out.  The problem is, if you (or someone else) has to refactor the code later on, this might turn into a subtle bug.)

Is it possible to create a new, empty `Cart` object here?  And if so, would that have length `0`?  Or do you need to have a user ID before you can make a cart?

(If this involves too much refactoring, feel free to mark it as a TODO and come back to it later.)

Reply:
I went with an early return here. Please see the changes in the commit 41f01790e9b01d56771328fb080c91592ba1618b and let me know what you think